tests/ios-podfile.test.ts

```diff
@@ -0,0 +1,128 @@
+jest.mock("danger", () => jest.fn());
+import danger from "danger";
+const dm = danger as any;
+
+import iosMacos from "../org/pr/ios-macos";
+
+// The mocked data and return values for calls the rule makes.
+beforeEach(() => {
+    dm.warn = jest.fn().mockReturnValue(true);
+    dm.message = jest.fn().mockReturnValue(true);
+    dm.fail = jest.fn().mockReturnValue(true);
+
+    dm.danger = {
+        github: {
+            pr: {
+                base: {
+                    ref: "",
+                },
+                head: {
+                    ref: "",
+                },
+            },
+            utils: {
+                fileContents: jest.fn(),
+            },
+            issue: {
+                labels: []
+            },
+        },
+        git: {
+            modified_files: [],
+            created_files: [],
+            deleted_files: []
+        }
+    };
+
+    dm.danger.github.utils.fileContents.mockReturnValue(Promise.resolve(""));
+});
+
+describe("Podfile should not reference commit hashes checks", () => {
+    it("fails when finds a commit reference", async () => {
+        dm.danger.github.utils.fileContents.mockReturnValueOnce(
+            Promise.resolve(
+                `DEPENDENCIES:
+                    - Gutenberg (from \`https://github.com/wordpress-mobile/gutenberg-mobile.git\`, tag \`v1.88.0\`)
+                    - TestPod(from \`https://github.com/test/pod.git\`, commit \`82f65c9cac2b59940db219a2327e12a0adfea4e3\`)
```

Comment:
[I just checked and the actual syntax in `Podfile.lock` has a space between the pod name and open parenthesis](https://github.com/wordpress-mobile/wordPress-iOS/commit/ab2cf5b07ce68947f781f7f9b1a28e80cc5fc73f#diff-e5b768ab2330ee29074f2eeadddd406f19e73b14673eb5f9b5ed6b5b77f4e363L557-R557) (as one would expect)

```suggestion
                    - TestPod (from \`https://github.com/test/pod.git\`, commit \`82f65c9cac2b59940db219a2327e12a0adfea4e3\`)
```